etc/cas.properties

```diff
@@ -32,6 +32,25 @@ google.analytics.id=
 google.analytics.autoLink=
 
 ##
+# OAuth: ORCID
+#
+oauth.orcid.authorize.url=https://orcid.org/oauth/authorize
+oauth.orcid.token.url=https://pub.orcid.org/oauth/token
+oauth.orcid.client.id=APP-FIAKWPX3JABTMOHK
+oauth.orcid.client.secret=6e585d6c-d70d-48df-825f-c13bf6739f49
```

Comment:
I'll be switching these to `${...:}` environment variables as you should never store even test secrets in a git commit.